src/main/java/BasketballGame/Game.java

```diff
@@ -0,0 +1,52 @@
+package BasketballGame;
+
+import java.time.LocalDateTime;
+
+public class Game {
+
+    //    private Team team;
+    private Team teamOne;
+    private Team teamTwo;
+    private int teamOneScore;
+    private int teamTwoScore;
+    private LocalDateTime gameTime;
+
+    public Game(Team teamOne, int teamOneScore, Team teamTwo, int teamTwoScore, LocalDateTime gameTime) {
+        this.teamOne = teamOne;
+        this.teamTwo = teamTwo;
+        this.teamOneScore = teamOneScore;
+        this.teamTwoScore = teamTwoScore;
+        this.gameTime = gameTime;
+    }
+
+    public static String getWinner(Game game) {
```

Comment:
What if I wanted to print a different message, but consistently in both cases? Avoid doing more than is asked- all that is asked is to return the winning team.
It should return the winning team or "Draw", nothing more.

Reply:
Now it returns only The winner or Draw